### DISCUSSION TITLE
Schema Config should have list of tables always visible

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5571] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
When configuring the schema in a database there is a lot of jumping between tables. Changing the base table with a selector in a separate dialog is slow and more difficult to use.

**Describe the solution you'd like**
A schema configurator that includes a list of all tables on the left.

<img width="1512" alt="image" src="https://user-images.githubusercontent.com/37256050/163608847-3ede9353-542f-4c7c-a592-c92e317941ee.png">

**Reported By**
grantfitzsimmons

**Comment by maxpatiiuk** [comment]
Displaying schema config for a single table requests hundreds of database records (one SpLocaleContainer; SpLocaleContainerItem per field, SpLocaleItemStr for each container item, container, for both name and description and for each language)

The difficulty with this is keeping track of unsaved changes to all the records and pushing changes to back end (but for modified records only as otherwise there would be too many requests)

**Comment by grantfitzsimmons** [comment]
This is still an awesome idea

**Comment by grantfitzsimmons** [comment]
Closed #3904 in favor of implementing this solution.
